contrib/ansible/openshift/README.md

```diff
@@ -0,0 +1,18 @@
+## Pbench Ansible
+Playbook to register pbench tools on openshift cluster
```

Comment:
Can you list what tools will be used for master nodes, non-master nodes, and compute nodes?